src/v1/events/controllers/eventController.js

```diff
@@ -26,4 +34,7 @@ class EventController {
   }
 }
 
-export default new EventController(eventDao);
+const eventController = new EventController(eventDao);
+setPermissions(eventController, methodPermissions);
```

Comment:
Also it's bad coz it mutates the passed parameter (controller) and this should be avoided when possible, so proper decorator will fix this

you then could do
```
const EnhancedEventController = permissionValidation(EventController);

export default new EnhancedEventController(eventDao);
```